client/client.js

```diff
@@ -0,0 +1,169 @@
+var assert = require('assert');
+var _ = require('lodash');
+var boot = require('loopback-boot');
+var debug = require('debug')('MeshModels.client');
```

Comment:
odd debug naming. is MeshModels is supposed to be a more readable strong-mesh-models?

and shouldn't the `.` be a `:`?

I think I'd stick with `strong-mesh-models:client`, it might be verbose, but it is very predictable, I think most of our code uses that pattern.